lib/screens/alcoholic/alcoholic_non_alcoholic_page.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class AlcoholicNonAlcoholicPage extends StatelessWidget {
+  const AlcoholicNonAlcoholicPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Alcoholic/Non Alcoholic'),
+      ),
+      body: const Center(
+        child: Text('Alcoholic/Non Alcoholic'),
```

Comment:
- [ ] Let's add this raw Strings in the Internationalization standard for future languages implementations